src/Registry/TypeRegistry.php

```diff
@@ -1198,23 +1206,29 @@ public function list_of( $type ) {
 	/**
 	 * Get the list of GraphQL type names to exclude from the schema.
 	 *
+	 * Type names are normalized using `strtolower()`, to avoid case sensitivity issues.
+	 *
 	 * @since @todo
 	 */
 	public function get_excluded_types() : array {
-		/**
-		 * Filter the list of GraphQL types to exclude from the schema.
-		 * 
-		 * Note: using this filter directly will NOT remove the type from being referenced as a possible interface or a union type.
-		 * To remove a GraphQL from the schema **entirely**, please use deregister_graphql_type();
-		 *
-		 * @param string[] $excluded_types The names of the GraphQL Types to exclude.
-		 *
-		 * @since @todo
-		 */
-		$excluded_types = apply_filters( 'graphql_excluded_types', [] );
+		if ( ! isset( $this->excluded_types ) ) {
```

Comment:
`! isset()` is future compatible with typed properties. (`null ===` will throw a ` Typed property must not be accessed before initialization` error)